essentials/javascript_objects_and_hijacking.mdx

```diff
@@ -209,3 +209,6 @@ Custom widget types can be added by providing a `getCustomWidgets` method in you
 Widgets can also be linked - the built in behavior of `seed` and `control_after_generate`, for example. 
 A linked widget has `.type = 'base_widget_type:base_widget_name'`; so `control_after_generate` may have
 type `int:seed`.
+
+## Prompt
+
```

Comment:
Empty heading?

Reply:
Working on this right now - happy to hold this PR back to wait for the next one